RELEASE-NOTES.md

```diff
@@ -4,6 +4,38 @@ section: overview
 releaseNoteTOC: true
 ---
 
+## 2020.02 release notes (2020-02-18)
+Packages released:
+
+### Components
+- **Data list:** Enabled hidden/visible feature to update display ([#2691](https://github.com/patternfly/patternfly-next/pull/2691))
+- **Data list, compact example:** Added compact version of data list ([#2686](https://github.com/patternfly/patternfly-next/pull/2686))
+- **Data toolbar:** 
+  - Updated input type to search ([#2648](https://github.com/patternfly/patternfly-next/pull/2648))
+  - Made Data toolbar experimental ([#2611](https://github.com/patternfly/patternfly-next/pull/2611))
+- **File upload:** Added File upload component ([#2681](https://github.com/patternfly/patternfly-next/pull/2681))
+- **Master detail:** Added Master detail demo ([#2645](https://github.com/patternfly/patternfly-next/pull/2645))
```

Comment:
```suggestion
- **Master detail:** Added master-detail demo ([#2645](https://github.com/patternfly/patternfly-next/pull/2645))
```

Reply:
I'm going to leave this alone until I hear back on what the name punctuation should be.